src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <div class="card__content">
+        <a href="#" class="card__image-product">
+          <img
+            src="./images/imac.jpeg"
+            class="card__image"
+            alt="Product">
+        </a>
+        <h1 class="card__title">
```

Comment:
It is better to use the h2 tag because this is a card, which means you can have a lot of them on your page, and h1 is recommended to use only 1 per 1 page